### DISCUSSION TITLE
configurazione itead dongle-e firmwere rcp multipan tramite zigbee2mqtt

### DISCUSSION THREAD

**Discussion by Calabash78** [indomus/forum#2641] [ZigBee, Z-Wave, Thread, Matter]
salve da giorni provo a migrare dalla coordinator cc2531 verso dongle-e come coordinator multipan rcp su zigbee2mqtt e hassOs versione 32 bit. su rasp 3b.
ho eseguito alla lettera le guide per installare come demone  il componente aggiuntivo silicon labs e quella di installare l'ultimo firmwere rcp sul dongle-e anche se no ho potuto installare matter mi dice che non e' supportato. 
quando avvio zigbee2mqtt mi rilascia questo errore:
```[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] socat.sh: executing... 
[18:23:37] INFO: Socat not enabled, marking service as down
[cont-init.d] socat.sh: exited 0.
[cont-init.d] zigbee2mqtt.sh: executing... 
[18:23:39] INFO: MQTT available, fetching server detail ...
[18:23:40] INFO: Previous config file found, checking backup
[18:23:40] INFO: Creating backup config in '/share/zigbee2mqtt/.configuration.yaml.bk'
[18:23:40] INFO: Adjusting Zigbee2mqtt core yaml config with add-on quirks ...
[cont-init.d] zigbee2mqtt.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[18:23:42] INFO: Handing over control to Zigbee2mqtt Core ...
> zigbee2mqtt@1.18.1 start /app
> node index.js
Zigbee2MQTT:info  2023-09-25 18:23:52: Logging to console and directory: '/share/zigbee2mqtt/log/2023-09-25.18-23-49' filename: log.txt
Zigbee2MQTT:info  2023-09-25 18:23:52: Starting Zigbee2MQTT version 1.18.1 (commit #1.18.1)
Zigbee2MQTT:info  2023-09-25 18:23:52: Starting zigbee-herdsman (0.13.71)
Zigbee2MQTT:error 2023-09-25 18:24:12: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2023-09-25 18:24:12: Failed to start zigbee
Zigbee2MQTT:error 2023-09-25 18:24:12: Exiting...
Zigbee2MQTT:error 2023-09-25 18:24:12: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
    at ZStackAdapter.<anonymous> (/app/node_modules/zigbee-herdsman/dist/adapter/z-stack/adapter/zStackAdapter.js:94:31)
    at Generator.throw (<anonymous>)
    at rejected (/app/node_modules/zigbee-herdsman/dist/adapter/z-stack/adapter/zStackAdapter.js:25:65)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zigbee2mqtt@1.18.1 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zigbee2mqtt@1.18.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2023-09-25T16_24_12_391Z-debug.log```
qualche suggerimento?grazie.

**Comment by KC-inDomus** [comment]
stai sicuramente sbagliando la configurazione di zigbee2mqtt, la sezione port è sbagliata

**Comment by Calabash78** [comment]
ciao per cosa intendi la sezione port sbagliata? seguendo la guida silicon labs nella parte di configurazione punto tcp://core-silabs-multiprotocol:9999
ma non riesco a mettere sotto "adapter ezsp" mi da un'errore

**Reply by indomus** [reply]
metti l'ip anzichè core-silabs-multiprotocol

**Reply by Calabash78** [reply]
ciao non ho capito se intendevi ip del raspberry come ip

**Comment by Calabash78** [comment]
quindi tcp://indirizzo raspberry:9999
corretto

**Reply by KC-inDomus** [reply]
sì

**Reply by Calabash78** [reply]
ciao ho provato ad inserire nell'addon Z2M  su `serial  port: tcp://192.168.x.x:9999` e non cambia nulla.
SE poi inserisco come `adapter: ezsp` non me lo fa inserire e mida un'errore ie mi dice che e' impossibile caricarlo.
mi chiedo se dipenda dalla installazione a 32 bit che ho di homeassistant Os in quanto non sono riuscito ad installare l'addon matter server.
uso un raspberry pi3b.
grazie

**Reply by indomus** [reply]
ricontrolla tutto, hai problemi su Silicon Labs Multiprotocol, se non ti risponde evidentemente non è salito in modo corretto. ciò detto, inutile mettere su tutto se non usi 64 bit (non potrai tirare su Matter, e quindi quel che stai facendo non ti serve a nulla).

**Comment by Manoel81** [comment]
Anch'io sono nella tua stessa situazione, stesso dongle, ma con Rpi4 e HASSIO a 64bit.

Quando vado ad avviare Z2M ottengo questo log:

[14:13:25] INFO: Preparing to start...
[14:13:26] INFO: Socat not enabled
[14:13:28] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:info  2023-10-13 14:13:35: Logging to console and directory: '/config/zigbee2mqtt/log/2023-10-13.14-13-35' filename: log.txt
Zigbee2MQTT:info  2023-10-13 14:13:35: Starting Zigbee2MQTT version 1.33.1 (commit #unknown)
Zigbee2MQTT:info  2023-10-13 14:13:35: Starting zigbee-herdsman (0.19.0)
Assertion failed: Command (setConfigurationValue) returned unexpected state: [object Object]
Assertion failed: Command (setValue) returned unexpected state: 55
Assertion failed: Command (setConfigurationValue) returned unexpected state: [object Object]
Assertion failed: Command (setValue) returned unexpected state: 55

Se provo ad andare su Z2M nella barra laterale, appare un riquadro che mi dice che il componente sembra non essere ancora pronto e si intravede la scritta "502 bad Gateway".

A differenza tua, però, la stringa "adapter: ezsp" sono riuscito ad includerla normalmente. Idee?

EDIT: riavviando HA, mi sono accorto che il dongle è stato rilevato come nuovo hardware e cliccando sul pulsante "configura" mi appare un errore "impossibile interrogare il dispositivo USB". Evidentemente ho qualche problema col dongle.

**Reply by Calabash78** [reply]
ciao ho risolto con una nuova installazione di hass Os. ho notato che certi confitti si sono creati su Zigbee2mqtt con il coordinator c2531
